manifests/prombench/benchmark/3c_promtail-test_deployment.yaml

```
@@ -0,0 +1,249 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: promtail-test
+  namespace: prombench-{{ .PR_NUMBER }}
+  labels:
+    app: promtail
+data:
+  promtail.yaml: |
+    client:
+      batchwait: 1s # Maximum wait period before sending batch
+      batchsize: 102400 # Maximum batch size to accrue before sending, unit is byte
+
+      timeout: 10s # Maximum time to wait for server to respond to a request
+
+      backoff_config:
+        minbackoff: 100ms # Initial backoff time between retries
+        maxbackoff: 5s # Maximum backoff time between retries
+        maxretries: 5 # Maximum number of retries when sending batches, 0 means infinite retries
+
+    server:
+      http_listen_port: 3101
+    positions:
+      filename: /run/promtail/positions.yaml
+    target_config:
+      sync_period: 10s # Period to resync directories being watched and files being tailed
+
+    scrape_configs:
+    - job_name: kubernetes-pods
+      kubernetes_sd_configs:
+      - role: pod
+      relabel_configs:
+      - action: keep
+        source_labels: [__meta_kubernetes_pod_label_app]
+        regex: prometheus|promtail
+      - source_labels: [__meta_kubernetes_pod_label_app]
+        target_label: __service__
+      - source_labels: [__meta_kubernetes_pod_node_name]
+        target_label: __host__
+      - action: replace
+        replacement: $1
+        separator: /
+        source_labels: [__meta_kubernetes_namespace, __service__]
+        target_label: job
+      - action: replace
+        source_labels: [__meta_kubernetes_namespace]
+        target_label: namespace
+      - action: replace
+        source_labels: [__meta_kubernetes_pod_name]
+        target_label: instance
+      - action: replace
+        source_labels: [__meta_kubernetes_pod_container_name]
+        target_label: container_name
+      - action: replace
+        source_labels: [__meta_kubernetes_pod_label_prometheus]
+        target_label: prometheus
+      - action: replace
+        source_labels: [__meta_kubernetes_pod_label_promtail]
+        target_label: promtail
+      - replacement: /var/log/pods/*$1/*.log
+        separator: /
+        source_labels: [__meta_kubernetes_pod_uid, __meta_kubernetes_pod_container_name]
+        target_label: __path__
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: promtail-test-pr-{{ .PR_NUMBER }}
```

If not necessary please remove all test string in the file content and filenames as this was a mistake when we started adding these configs.
```suggestion
  name: promtail-pr-{{ .PR_NUMBER }}
```

where nececary maybe replace with `benchtest`. for me just `test` sound more like some debug code.

---

okay will add this change, maybe can add another pr to remove the other `test-`s later.
I think it's related to https://github.com/prometheus/prombench/pull/89

---

yes that would be great, and can probably remove prow mentions everywhere as well.